pl_bolts/losses/giou.py

```diff
@@ -0,0 +1,38 @@
+"""
+Generalized Intersection over Union (GIoU) loss (Rezatofighi et. al)
+"""
+
+import torch
+
+
+def giou_loss(pred: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
+    """
+    Calculates the generalized intersection over union loss.
+
+    Args:
+        pred: batch of prediction bounding boxes with representation
+              [x_min, y_min, x_max, y_max]
+        target: batch of target bounding boxes with representation
+                [x_min, y_min, x_max, y_max]
+
+    Returns:
+        loss
```

Comment:
rather be `GIoU` loss instead of `loss`?
```suggestion
        GIoU loss
```